config/routes.rb

```diff
@@ -40,23 +40,12 @@
   get "/service-moved", to: "maintenance#service_moved"
   get "/service-unavailable", to: "maintenance#service_unavailable"
 
-  get "/download-23-24-lettings-form", to: "start#download_23_24_lettings_form"
-  get "/download-23-24-lettings-bulk-upload-template", to: "start#download_23_24_lettings_bulk_upload_template"
-  get "/download-23-24-lettings-bulk-upload-legacy-template", to: "start#download_23_24_lettings_bulk_upload_legacy_template"
```

Comment:
Remove legacy specific paths, cause there shouldn't be anywhere we link to 2023 resources anymore. Refactored the rest